firefox/manifest.json

```diff
@@ -11,21 +11,19 @@
     }
   },
   "icons": {
-    "16": "dist/img/favicon/favicon-16x16.png",
-    "32": "dist/img/favicon/favicon-32x32.png",
-    "64": "dist/img/favicon/favicon-64x64.png",
-    "96": "dist/img/favicon/favicon-96x96.png",
-    "128": "dist/img/favicon/favicon-128x128.png"
+    "32": "dist/img/rested-logo.svg"
   },
   "browser_action": {
     "default_title": "Open RESTED tab",
-    "default_icon": {
-      "16": "dist/img/favicon/favicon-16x16.png",
-      "32": "dist/img/favicon/favicon-32x32.png",
-      "64": "dist/img/favicon/favicon-64x64.png",
-      "96": "dist/img/favicon/favicon-96x96.png",
-      "128": "dist/img/favicon/favicon-128x128.png"
-    }
+    "browser_style": true,
```

Comment:
Any reason you would need to add `browser_style`? I don't see you using any `.browser-style` classes here.

https://developer.mozilla.org/en-US/Add-ons/WebExtensions/user_interface/Browser_styles

Reply:
Sorry. My mistake. I was testing some other things in order to display the right icon color.